recipes/kirigami

```diff
@@ -0,0 +1,4 @@
+(kirigami
+ :fetcher github
+ :repo "jamescherti/kirigami.el"
+ :files (:defaults "extensions/kirigami*.el"))
```

Comment:
Just making a note (you may already be aware) that there is no `extensions` directory in the repo.